website/docs/cdktf/concepts/modules.mdx

```diff
@@ -87,6 +87,10 @@ For local modules, use the object format.
 }
 ```
 
+#### Generating bindings for submodules
+
+For performance reasons we don't generate bindings for submodules. To generate bindings for submodules, need to specify the module source as `terraform-aws-modules/eks/aws//modules/eks-managed-node-group`, where after the `//` is the path to the submodule in the modules repository.
```

Comment:
Could we try putting this in the same section with "Generating Bindings for modules"?  I'm not sure it needs its own subsection. 

```suggestion
For performance reasons we don't automatically generate bindings for submodules. To generate bindings for submodules, specify the module source as `terraform-aws-modules/eks/aws//modules/eks-managed-node-group`, where after the `//` is the path to the submodule in the modules repository.
```